Assets/PurrNet/Runtime/Components/NetworkPrefabs/IPrefabProvider.cs

```diff
@@ -3,25 +3,33 @@
 
 namespace PurrNet
 {
+    public struct PrefabData
+    {
+        public int prefabId;
+        public GameObject prefab;
+        public bool pooled;
+        public int warmupCount;
+    }
+
     public interface IPrefabProvider
     {
-        IReadOnlyList<NetworkPrefabs.PrefabData> allPrefabs { get; }
-        
-        bool TryGetPrefab(int id, out GameObject prefab);
-        
-        bool TryGetPrefabData(int id, out NetworkPrefabs.PrefabData prefab);
-        
-        bool TryGetPrefab(int id, int offset, out GameObject prefab);
+        IEnumerable<PrefabData> allPrefabs { get; }
+
+        bool TryGetPrefabData(int prefabId, out PrefabData prefabData);
+
+        bool TryGetPrefabData(GameObject prefab, out PrefabData prefabData);
+
+        bool TryGetPrefab(int prefabId, int offset, out GameObject prefab);
```

Comment:
As far as I can tell this method is not actually used anywhere and could possibly be removed.